Sources/AppStoreConnectCLI/Commands/TestFlight/BetaTesters/ListBetaTestersCommand.swift

```diff
@@ -13,98 +13,66 @@ struct ListBetaTestersCommand: CommonParsableCommand {
     @OptionGroup()
     var common: CommonOptions
 
-    @Option(help: "The bundle ID of an application. (eg. com.example.app)")
-    var filterBundleId: String?
-
-    @Option(help: "The name of the beta group")
-    var filterGroupName: String?
-
-    @Option(default: 10,
-            help: "Number of included related resources to return.")
-    var relatedResourcesLimit: Int
-
-    private enum ListBetaTesterError: LocalizedError {
-        case multipleFilters
-
-        var failureReason: String? {
-            switch self {
-                case .multipleFilters:
-                    return "Only one relationship filter can be applied"
-            }
-        }
-    }
-
-    private enum ListStrategy {
-        case all
-        case listByApp(bundleId: String)
-        case listByGroup(betaGroupName: String)
-        case listByAppAndGroup
-
-        typealias ListOptions = (bundleId: String?, betaGroupName: String?)
-
-        init(options: ListOptions) {
-            switch (options.bundleId, options.betaGroupName) {
-                case let(.some(bundleId), .some(betaGroup)) where !bundleId.isEmpty && !betaGroup.isEmpty:
-                    self = .listByAppAndGroup
-                case let(.some(bundleId), _) where !bundleId.isEmpty:
-                    self = .listByApp(bundleId: bundleId)
-                case let(_, .some(betaGroupName)) where !betaGroupName.isEmpty:
-                    self = .listByGroup(betaGroupName: betaGroupName)
-                case (_, _):
-                    self = .all
-            }
+    @Option(help: "Beta tester's email.")
+    var filterEmail: String?
+
+    @Option(help: "Beta tester's first name.")
+    var filterFirstName: String?
+
+    @Option(help: "Beta tester's last name.")
+    var filterLastName: String?
+
+    @Option(
+        help: """
+        An invite type that indicates if a beta tester was invited by an email invite or used a TestFlight public link to join a beta test. \n
+        Possible values \(BetaInviteType.allCases).
+        """
+    ) var filterInviteType: BetaInviteType?
+
+    @Option(
+        parsing: .upToNextOption,
+        help: "Application Bundle Ids. (eg. com.example.app)"
+    ) var filterApps: [String]
```

Comment:
I have been using `filterBundleIds` because `filterApps` could be ambiguous
What do you think?

I understand this deviates from the issue, maybe @orj has input here

You can look at `ListBetaGroupsCommand` for an example of allowing both bundle ids and app ids
I believe @orj has all allowed both for `ListPreReleaseVersionsCommand` in #121 as well